src/cli/build.rs

```diff
@@ -0,0 +1,26 @@
+use clap::Args;
+
+use super::{Handler, Result};
+
+/// Arguments for `BuildCommand`
+///
+/// Usage :
+/// `quark build --image <IMAGE>`
+#[derive(Debug, Args)]
+pub struct BuildCommand {
+    #[clap(short, long)]
```

Comment:
Please add comments to all your options, to explain what they're for. This is what will appear with the `--help` option.